core/src/saros/editor/colorstorage/ColorIDSetStorage.java

```diff
@@ -182,12 +177,15 @@ private void load() {
     if (serializedData.isEmpty()) return;
 
     try {
-      currentAvailableSets =
-          (List<ColorIDSet>)
-              new ObjectInputStream(new ByteArrayInputStream(serializedData.getBytes(CHARSET)))
-                  .readObject();
+      currentAvailableSets = (List<ColorIDSet>) createXStream().fromXML(serializedData);
     } catch (Exception e) {
       LOG.error("error while loading color sets", e);
     }
   }
+
+  private XStream createXStream() {
+    XStream xStream = new XStream(new DomDriver(CHARSET));
```

Comment:
Is this the same behavior as before ? To be more specific, is there any need for a charset at all ? Because you are producing already a string.

Reply:
I will check this. Thank you, for the review

Reply:
Done